bots/tests-scan

```diff
@@ -149,20 +162,43 @@ def tests_invoke(priority, name, revision, ref, context, base, options):
         return
     current = time.strftime('%Y%m%d-%H%M%M')
 
-    checkout = "PRIORITY={priority:04d} bots/make-checkout --verbose"
-    cmd = "TEST_NAME='{name}-{current}' TEST_REVISION='{revision}' bots/tests-invoke"
-    if base:
-        cmd += " --rebase='{base}'"
-        checkout += " --base='{base}'"
+    header = "PRIORITY={priority:04d} "
+
+    if not options.module_repo:
+        checkout = "bots/make-checkout --verbose"
+        cmd = "TEST_NAME='{name}-{current}' TEST_REVISION='{revision}' bots/tests-invoke"
 
-    if options.repo:
-        checkout += " --repo='{repo}'"
-        cmd = "GITHUB_BASE='{repo}' " + cmd + " --remote='test'"
+        if base:
+            cmd += " --rebase='{base}'"
+            checkout += " --base='{base}'"
 
-    cmd += " '{context}' '{ref}'"
-    checkout += " '{ref}' '{revision}' && "
+        if options.repo:
+            checkout += " --repo='{repo}'"
+            cmd = "GITHUB_BASE='{repo}' " + cmd + " --remote='test'"
 
-    return (checkout + cmd).format(
+        cmd += " '{context}' '{ref}'"
+        checkout += " '{ref}' '{revision}' && "
+
+    else:
+        # When testing new versions of external modules, we run the
+        # regular Cockpit tests in a regular Cockpit master checkout.
+        # The 'updater' will use TEST_HEAD_REPO and TEST_HEAD_REF to
+        # copy the new external module into that checkout.  These
```

Comment:
This is a bit confusing, as the npm updater script doesn't actually copy anything; maybe "... to adjust the Cockpit checkout to use the new external module"?